include/xtensor/xoffset_view.hpp

```diff
@@ -26,11 +26,43 @@ namespace xt
             using pointer = M*;
             using const_pointer = const M*;
 
+            using proxy = xtl::xproxy_wrapper<M>;
+
+            template <class value_type, class requested_type>
+            using simd_return_type = xsimd::simd_return_type<value_type, requested_type>;
+
             template <class T>
             decltype(auto) operator()(T&& t) const
             {
                 return xtl::forward_offset<M, I>(t);
             }
+
+            // template <class align, class requested_type, std::size_t N, class E>
+            // auto proxy_simd_load(const E& expr, std::size_t n) const
+            // {
+            //     using simd_value_type = xsimd::simd_type<value_type>;
+            //     auto v1 = xsimd::load_aligned((double*) expr.data() + n);
+            //     auto v2 = xsimd::load_aligned((double*) expr.data() + n + N);
+
+            //     if (std::is_same<M, double>::value && I == sizeof(double))
+            //     {
+            //         return simd_value_type(_mm256_permute4x64_pd(_mm256_unpackhi_pd(v1, v2), _MM_SHUFFLE(3, 1, 2, 0)));
+            //     }
+            //     else if (std::is_same<M, double>::value && I == sizeof(double))
+            //     {
+            //         return simd_value_type(_mm256_permute4x64_pd(_mm256_unpacklo_pd(v1, v2), _MM_SHUFFLE(3, 1, 2, 0)));
+            //     }
+            //     // return expr.template load_simd<align, double, N>(n);
+            // }
+
+            // template <class align, class simd, class E>
+            // auto proxy_simd_store(E& expr, std::size_t n, const simd& batch) const
+            // {
+            //     using simd_value_type = typename E::simd_value_type;
+            //     // return expr.template store_simd<align>(n, xsimd::select(batch, simd_value_type(0), simd_value_type(1)));
+            //     return 0;
+            // }
```

Comment:
What is this meant for?

Reply:
This is to allow `load_simd` and `store_simd` to work on offset views.

Reply:
I've implemented it now and added tests. This works pretty great in combination with xsimd complex support.